site/src/qa_info_bar/QaBar.jsx

```diff
@@ -95,4 +150,3 @@ export default function QaBar({viewState, setViewState, activeOsmFeature, setAct
     viewState: PropTypes.object.isRequired,
     setViewState: PropTypes.func.isRequired,
   };
-  
```

Comment:
Revert?

Reply:
I think the original has an extra newline at the end so I'm just getting rid of that.

Reply:
Actually they were two extra spaces that are now removed